### DISCUSSION TITLE
History function

### DISCUSSION THREAD

**Discussion by broland07** [PrivateBin/PrivateBin#1239] [Q&A] [answered]
### Describe the problem/question

Hi guys!

I use privatebin for my own internal network. Is it possible to log all the paste and list them? For example if i want to check the 2 days earlier pastes is it possible?

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

I checked on the dashboard but i didn't find it.

### Server OS

Ubuntu

### PrivateBin version

v 1.6.2

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by rugk** [comment, marked as answer]
> Is it possible to log all the paste and list them?

So you want to see all pates that have been created?

Of course, you cannot see the content – after all, that's the whole purpose of the project. But you may see what has been created in encrypted form.

If you save it on the filesystem, the directory/file (path) is the paste ID.
In the DB it is just inside the DB effectively.
